### ISSUE TITLE
Would consider supporting Folia?

### ISSUE THREAD

**Issue by CuleSKY** [brunyman/MPDB#108]
Would this plugin consider to support Folia?

**Comment by brunyman** [comment]
Yes, we will at some point, currently this uses the single threaded design as a event trigger order so it will need some changes.

**Comment by xcomcadx** [comment]
when could we be seeing support for folia?

**Comment by xcomcadx** [comment]
my server heavily relies on this plugin and i am currently in the middle of switching to folia
